pulsar-client/src/main/java/org/apache/pulsar/client/impl/conf/ClientConfigurationData.java

```diff
@@ -327,6 +333,14 @@ public boolean isUseTls() {
         return false;
     }
 
+    public long getLookupTimeoutMs() {
+        if (lookupTimeoutMs >= 0) {
+            return lookupTimeoutMs;
+        } else {
+            return operationTimeoutMs;
+        }
+    }
```

Comment:
It's better to add extra docs to note this behavior since the default `lookupTimeoutMs` is -1, users may think it means the lookup request never time out.

Reply:
Add extra docs where? ClientBuilder already documents that lookupTimeout defaults to operationTimeout

Reply:
@BewareMyPower ^

Reply:
@BewareMyPower https://github.com/apache/pulsar/pull/11627/files#diff-33219b2b08e58d5841feb88fdae064889f2477fd77564044ff4bb4e8fc24f1f9R220

Reply:
Thanks

Reply:
@BewareMyPower you're still marked as "Change requested"